### PR TITLE
Fix of a problem with invariant 4 in the qtreetbl

### PR DESCRIPTION
**Pull request by darmar-lt**
Patch ads a test case for the invariant 4 in the qtreetbl and fixes the problem.

### CODE REVIEW

File: src/containers/qtreetbl.c [hunk 1]
```diff
@@ -1050,6 +1195,11 @@ static qtreetbl_obj_t *put_obj(qtreetbl_t *tbl, qtreetbl_obj_t *obj,
     if (is_red(obj->left) && is_red(obj->left->left)) {
         obj = rotate_right(obj);
     }
+    
+    // split 4-nodes on the way up.
+    /*if (is_red(obj->left) && is_red(obj->right)) {
```
Comment (wolkykim) [inline src/containers/qtreetbl.c]: can we remove this if this is not needed?

**Comment by wolkykim** [comment]
thank you for the update. left a minor comment.

File: src/containers/qtreetbl.c (hunk 1 above)
Comment (darmar-lt) [reply src/containers/qtreetbl.c]: I removed these comments in the last commit.

**Comment by wolkykim** [comment]
Thanks I am reviewing the change.

**Pull request merged**